packages/sewing-kit-koa/src/tests/assets.test.ts

```diff
@@ -456,19 +505,27 @@ function mockEntrypoint({
   return {js: scripts, css: styles};
 }
 
-function mockManifest(
-  entrypoints: {[key: string]: Entrypoint} = {},
-  asyncAssets: {[key: string]: AsyncAsset[]} = {},
-) {
-  return {entrypoints, asyncAssets};
+function mockManifest({
```

Comment:
The manifest doesn't just contain the `entrypoints` and `asyncAssets` anymore. Updating this helper to just take in an object of type `Manifest` instead of piecing in the params together.